droidlet/perception/robot/handlers/label_propagate.py

```diff
@@ -2,21 +2,57 @@
 Copyright (c) Facebook, Inc. and its affiliates.
 """
 from .core import AbstractHandler
-from tokenize import String
 import numpy as np
 import os
-import sys
-if "/opt/ros/kinetic/lib/python2.7/dist-packages" in sys.path:
-    sys.path.remove("/opt/ros/kinetic/lib/python2.7/dist-packages")
 import cv2
 import json
-from copy import deepcopy as copy
-from IPython import embed
-import time
-from scipy.spatial.transform import Rotation
 import glob
-import argparse
 from droidlet.lowlevel.robot_mover_utils import transform_pose
+from numba import njit
+from math import ceil, floor
+
+def compute_uvone(height, width):
+    intrinsic_mat = np.array([[256, 0, 256], [0, 256, 256], [0, 0, 1]])
```

Comment:
nit: can these be constants?  also, can they be formatted ?  e.g. outside of the function, do

INTRINSIC_MAT = np.array([[256, 0,      256 ], 
                                                 [0,     256,  256 ], 
                                                 [0,     0,       1      ]])
                                                 
     and say what each of these constants is with a comment

Reply:
lol github code blocks do not preserve the formatting.  left edge of numbers should be aligned

Reply:
resolved in the comment below.